devops/actions/e2e-tests-win/action.yml

```diff
@@ -32,6 +32,9 @@ runs:
   - name: Configure
     shell: cmd
     run: |
+      echo "::group::sycl-ls --verbose"
+      sycl-ls --verbose
+      echo "::endgroup::
       mkdir build-e2e
       set PATH=%GITHUB_WORKSPACE%\install\bin;%PATH%
```

Comment:
Don't think it can be earlier than `%PATH%` setting...

Reply:
Oh, yes. Didn't noticed that...